CMakeLists.txt

```diff
@@ -4,54 +4,62 @@ project(darwin)
 
 set(CMAKE_VERBOSE_MAKEFILE on)
 set(CMAKE_CXX_STANDARD 17)
-set(ENV{PKG_CONFIG_PATH}  "/usr/local/lib/pkgconfig/;/usr/local/libdata/pkgconfig/")
-
-find_package(PkgConfig)
 
 if (DEBUG)
     SET(CMAKE_CXX_FLAGS "-W -Wall -Wextra -g -ggdb3")
 else ()
     SET(CMAKE_CXX_FLAGS "-W -Wall -Wextra")
 endif ()
 
-message("Flags: ${CMAKE_CXX_FLAGS}")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
+message("${CMAKE_MODULE_PATH}")
+
+if (NOT DEFINED FILTER)
+    set(
+        FILTERS
+        HOSTLOOKUP
+        SESSION
+        DGA
+        CONNECTION
+        TANOMALY
+        ANOMALY
+        LOGS
+        CONTENT_INSPECTION
+    )
+else (NOT DEFINED FILTER)
```

Comment:
je connais pas très bien la syntaxe CMake, d'après la doc ça a l'air bon mais c'est étrange
c'est bien NOT DEFINED ?